scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```diff
@@ -36,7 +37,7 @@ RUN apt-get update; \
     apt-get install -qy python3-pip;
 
 # Install cmake 3.21.2 (minimum requirement is cmake 3.10)
-RUN wget https://github.com/Kitware/CMake/releases/download/v3.21.2/cmake-3.21.2-Linux-x86_64.sh; \
+RUN wget -q https://github.com/Kitware/CMake/releases/download/v3.21.2/cmake-3.21.2-Linux-x86_64.sh; \
     test "$(sha256sum cmake-3.21.2-Linux-x86_64.sh)" = "3310362c6fe4d4b2dc00823835f3d4a7171bbd73deb7d059738494761f1c908c  cmake-3.21.2-Linux-x86_64.sh"; \
     chmod +x cmake-3.21.2-Linux-x86_64.sh; \
     ./cmake-3.21.2-Linux-x86_64.sh --skip-license --prefix="/usr"
```

Comment:
@bshastry Do we even need this? I see that focal ships cmake 3.16, which should work according to the comment above. So perhaps we could remove this and simplify the image.